.github/workflows/deploy-release.yml

```diff
@@ -151,6 +152,28 @@ jobs:
             ${{ steps.get-name.outputs.full-name }}.tar.gz
             ${{ steps.get-name.outputs.full-name }}.conda-lock.yml
 
+  create-git-tag:
+    name: Create Git Tag
+    runs-on: ubuntu-latest
+    needs: [update-dependencies, set-version]
+    # Run independently from the event if none of the needed jobs failed
+    if: ${{ !cancelled() && needs.update-dependencies.result == 'success' && needs.set-version.result == 'success' }}
+    env:
+      VERSION: ${{ needs.set-version.outputs.version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          token: ${{ secrets.REPO_CONTENT_WRITE_TOKEN }}
+          fetch-tags: true
+
+      - name: Create and push tag
+        shell: bash -el {0}
+        run: |
```

Comment:
Before this job, if you intend to use `access-bot` as the committer, you should import the gpg key associated with that account. It'll also set all the required `git` stuff so the below command will work. See https://github.com/ACCESS-NRI/dev-docs/wiki/Continuous-integration-and-Deployment#verified-commits-from-a-workflow

Reply:
I think the committer should already be `access-bot` because I am using `${{ secrets.REPO_CONTENT_WRITE_TOKEN }}` (which references a token created by `access-bot`) in the checkout step.

I would still need to add the GPG action for signing the tag though, so I will add it anyway.

Reply:
Fixed in [857f983b9206c20f187a9058444f54477b8f6807](https://github.com/ACCESS-NRI/access-ram-condaenv/pull/13/commits/857f983b9206c20f187a9058444f54477b8f6807)